changelog/fragments/1672911999-v2-health-status.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: enhancement
+
+# Change summary; a 80ish characters long description of the change.
+summary: | 
+  Health Status: Elastic Agent now indicates detailed status information for each component/unit.
```

Comment:
a lot of decribing, should we produce a blog post we can refer to?

Reply:
In addition to the changelog entries, I will update our documentation and be more verbose there.

Reply:
> a lot of decribing, should we produce a blog post we can refer to?

Yeah a blog post is a good idea, it will be a lot easier to summarize the major changes there.